### PR TITLE
fix tut1 typos and grammar

### PR DESCRIPTION
**Pull request by Affie**

### CODE REVIEW

File: julia/caesar-api/icra-1-simple.ipynb [hunk 1]
```diff
@@ -3878,7 +3878,7 @@
     "\n",
     "The figure shows the position and orientation (red forward) for poses `x0` and `x1`. As well as the covariance ellipse. \n",
     "Since the solver used was non-parametric, the covariance ellipse is based on a best Gaussian distribution fit of the full belief.\n",
-    "A few other functions are also handy for interacting with the factor graph, for instance `getVariable` returns the full variable.\n",
+    "A few other functions are also handy for interacting with the factor graph, for instance, `getVariable` returns the full variable.\n",
```
Comment (GearsAD) [inline julia/caesar-api/icra-1-simple.ipynb]: I don't think there's a second comma here actually

**Review by GearsAD — APPROVED** [review]

File: julia/caesar-api/icra-1-simple.ipynb (hunk 1 above)
Comment (Affie) [reply julia/caesar-api/icra-1-simple.ipynb]: ok, I removed it

**Comment by GearsAD** [comment]
Thanks for sorting out all the speeling mistakes! I missed all of those

**Comment by Affie** [comment]
A lot of them were my fingers being dumb and typing out of order.

**Pull request merged**